linux/deploy_vm/rebuild_unattend_install_iso.yml

```diff
@@ -283,3 +283,14 @@
             dest_file: "md5sum.txt"
       when: not ubuntu_bios_cfg_exist
   when: unattend_install_conf is match('Ubuntu/Server')
+
+- name: "Rebuild ISO for FreeBSD"
+  block:
```

Comment:
Block is not needed if there's only one task in it.

Reply:
Change it. Thanks for your suggestion.